File: app/src/main/java/com/gmail/fattazzo/formula1world/ergast/imagedb/objects/DriverConstructor.java
package com.gmail.fattazzo.formula1world.ergast.imagedb.objects;

import com.activeandroid.Model;
import com.activeandroid.annotation.Column;
import com.activeandroid.annotation.Table;

/**
 * @author fattazzo
 *         <p/>
 *         date: 05/06/17
 */
@Table(name = "driversConstructors")
public class DriverConstructor extends Model {

    @Column
    long driverId;

    @Column
    long constructorId;

    @Column
    int year;
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/ergast/imagedb/objects/DriverConstructor.kt
package com.gmail.fattazzo.formula1world.ergast.imagedb.objects

import com.activeandroid.Model
import com.activeandroid.annotation.Column
import com.activeandroid.annotation.Table

/**
 * @author fattazzo
 *
 *
 * date: 05/06/17
 */
@Table(name = "driversConstructors")
class DriverConstructor : Model() {

    @Column
    internal var driverId: Long = 0

    @Column
    internal var constructorId: Long = 0

    @Column
    internal var year: Int = 0
}
